File: src/main/java/edu/marmara/dto/ScheduleGetDTO.java
package edu.marmara.dto;

import java.util.List;

public class ScheduleGetDTO {
    private List<String> courses;

    public ScheduleGetDTO(List<String> courses) {
        this.courses = courses;
    }

    public ScheduleGetDTO() {
    }

    public List<String> getCourses() {
        return courses;
    }

    public void setCourses(List<String> courses) {
        this.courses = courses;
    }

    public void addCourse(String course){
        this.courses.add(course);
    }

    @Override
    public String toString() {
        return "ScheduleGetDTO{" +
                "courses=" + courses +
                '}';
    }
}

File: python/src/main/marmara/dto/ScheduleGetDTo.py
class ScheduleGetDTO:
    def __init__(self, courses=None, approved=None, send_to_review=None):
        self.courses = courses or []
        self.approved = approved
        self.send_to_review = send_to_review

    def add_course(self, course):
        self.courses.append(course)

    def __str__(self):
        return f"ScheduleGetDTO{{courses={self.courses}}}"
